packages/kbn-text-based-editor/src/esql_documentation_sections.tsx

```diff
@@ -89,15 +88,15 @@ ES|QL can access the following metadata fields:
 Use the \`METADATA\` directive to enable metadata fields:
 
 \`\`\`
-FROM index [METADATA _index, _id]
```

Comment:
ℹ️ Irrelevant with the change but I decided to fix the docs for metadata in this PR too